tutorial/tutorial04-PressureVsTe/Lymberopoulos_paper.geo

```diff
@@ -0,0 +1,187 @@
+//dom0Mult = 1/25.4e-3;
+dom0Mult = 1;
```

Comment:
Is there a reason you leave the commented prior value in? If so, I would add a small comment as to why. Otherwise, probably should remove it.

Reply:
No, this is just an unscaled version of a previous mesh.